FusionIIIT/applications/hostel_management/models.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+from django.db import models
+from django.contrib.auth.models import User
+from applications.globals.models import ExtraInfo, Staff
+from applications.academic_information.models import Student
+from applications.complaint_system.models import Caretaker
+from django.utils import timezone
+
+
+class HostelManagementConstants:
+    ROOM_STATUS = (
+        ('Booked', 'Booked'),
+        ('CheckedIn', 'CheckedIn'),
+        ('Available', 'Available'),
+        ('UnderMaintenance', 'UnderMaintenance'),
```

Comment:
The value of the second element is what will be visible on the UI. So, it should be in normal English and not in camel case. "Unter Maintainance" and not "UnderMaintainance".

Same for the values in BOOKING_STATUS.